### ISSUE TITLE
A11y_code.visualstudio.com_API Working With Extensions_Continuous Integration_Keyboard: "Hash" button in front of any heading is not accessible with keyboard.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#3709]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS2.1.1;#Win10;#Visual Studio Code;#Keyboard; #CAITestPass_May2020;#FTP;#STP;

**Environment Details**:
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

**Repro Steps**:
1. Launch the application URL :https://code.visualstudio.com
2. Navigate to "API" link.
3. From left navigation, select "Working With Extensions".
4. Navigate to "Continuous Integration" link.
5. Navigate to # Hash button in front of any heading.

**Actual**:
"Hash" # button in front of any heading is not accessible with keyboard.

**Similar issue observed in through out the application where "Hash" # button is present**.

**Expected**:
All actionable control should be accessible using keyboard.

**Attachment link for Reference**:
![3709_A11y_code visualstudio com_API Working With Extensions_Continuous Integration_Keyboard Hash button in front of any heading is not accessible with keyboard](https://user-images.githubusercontent.com/52035395/83004374-374d3280-a02d-11ea-96f0-bc9d218fb077.gif)

**Comment by JacksonKearl** [comment]
Thanks for reporting! Made them transparent instead of hidden. This means they'll be selected on copying/paste, but that's not a terribly big deal IMO.

![sdfasfg](https://user-images.githubusercontent.com/8586769/87488726-f23d8980-c5f5-11ea-8d66-d285e6b24c66.gif)

**Comment by PoojaNamde** [comment]
@JacksonKearl We are not able to find similar control in the https://code.visualstudio.com/api, could you please help

**Comment by 12-shweta** [comment]
#Closed;
